### ISSUE TITLE
UX for providing help for markdown syntax 

### ISSUE THREAD

**Issue by Preeticp** [openshiftio/openshift.io#3145]
Related to #3065.
As an immediate solution, a note with required links has been added to the docs. (https://docs.prod-preview.openshift.io/user-guide.html#adding_description_to_a_work_item)

@sunilmalagi, as per @xcoulon a direct help link from Description/Comments section in the UX to Markdown documentation would help. Can you please explore further?

**Comment by sunilmalagi** [comment]
@nimishamukherjee @michaelkleinhenz @Preeticp I would like to know if you want to implement suggestion made by Xavier.  

If yes, I can provide 2 options, they are: 
Option 1:  Provide a link for help which opens up https://docs.prod-preview.openshift.io/user-guide.html#adding_description_to_a_work_item 

Option2: Provide 2 links saying  "Markdown Syntax  &  Extensions" which opens 2 different pages respectively 
a. https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet   OR 
b. https://github.com/russross/blackfriday#extensions

What is that User would like to see?  Option 1 will have 2 click to reach the Cheatsheet/Extensions, and option 2 will get the direct pages.  I will work on the mockups once you confirm on the same.

**Comment by Preeticp** [comment]
@sunilmalagi if you go ahead on this, please note that this link: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet should be replaced by Red Hat's mark down help page: https://access.redhat.com/help/markdown.

**Comment by sunilmalagi** [comment]
@nimishamukherjee @michaelkleinhenz @Preeticp @xcoulon 
here is the visuals for "markdown syntax", there are 2 options, please check it out and let me know for any feedback: 
https://redhat.invisionapp.com/share/R8I1JRXFVGQ#/295251624_00_Markdown_Syntax

**Comment by michaelkleinhenz** [comment]
@Preeticp @sunilmalagi please note that markdown isn't markdown and different implementations are possibly (and in reality) a bit different here and there. I received some complaints about our markdown not doing this or that in the last days, but we can't really do something there as we're using an upstream markdown library. 

Here's the library we're using: https://github.com/russross/blackfriday

**Comment by Preeticp** [comment]
@michaelkleinhenz you are right and currently our docs have a note with the links to the official Red Hat page as well as the blackfriday repo here:  https://docs.prod-preview.openshift.io/user-guide.html#adding_description_to_a_work_item - "In addition to the [standard markdown syntax](https://access.redhat.com/help/markdown), refer to [these extensions](https://github.com/russross/blackfriday#extensions) for more information about the syntax."

**Comment by Veethika** [comment]
@Preeticp @nimishamukherjee @sunilmalagi has this been implemented already?

**Comment by Veethika** [comment]
Unassigning myself as UX solution has been provided. Please assign again if needed. @Preeticp also please update on the status of this issue.

**Comment by Preeticp** [comment]
@michaelkleinhenz @nimishamukherjee currently I see only 'Markdown supported' mentioned in the UX. Is this how we decided to implement this or are we implementing the suggestion made by @sunilmalagi? IMHO the second alternative linking to Markdown syntax is better, however I am not sure if we want two links there or only one. If we decide on one, which one: the RH .md help page or the black friday extensions link used in OSIO.

**Comment by sunilmalagi** [comment]
I am removing area/UX & team/ux labels as I have provided the solutions.

**Comment by joshuawilson** [comment]
@sunilmalagi This task has `UX` in the title asking for your help. It is not about implementing a solution. If you have don everything needed, then you should verify with @Preeticp and close it. If we need to implement something then a new task can be created and linked to this one.

**Comment by nimishamukherjee** [comment]
@sunilmalagi https://redhat.invisionapp.com/share/R8I1JRXFVGQ#/295251624_00_Markdown_Syntax
has various options.
Provide the updated invision link with the final option.

**Comment by sunilmalagi** [comment]
@nimishamukherjee I have provided 2 options to review, I did review it with team and we couldn't decide on any, I am not sure about the reason.

Option 1: 'Markdown supported' link takes you to the page, where the user will land on the page with 'Markdown Syntax & Extensions" links.. 

Option 2: is having 'Markdown Syntax & Extensions" links upfront and which takes you directly to the respective pages.  

May be we need to check with @Preeticp and take a call on the same.  Once we decide on it, I will retain the one which is approved.

**Comment by sunilmalagi** [comment]
@Preeticp @nimishamukherjee 
Here is the updated visuals for #3145, I have retained the 2nd option as suggested by Preeti and made the changes for "Markdown syntax and extensions"
https://redhat.invisionapp.com/share/83I1JRFM2AR#/295251625_000_Markdown_Syntax

**Comment by sunilmalagi** [comment]
@Preeticp @nimishamukherjee can we take out the 'team ux' label if the issue got solved? I will also un assign myself when you confirm me.

**Comment by christianvogt** [comment]
@sunilmalagi once ux is complete with their deliverables in the issue, update the label to assign to the ui team.

I find it odd that our docs send the user to another dev site to find additional information about our support. This means that we expect the external reference to remain unchanged and if don't keep our library up to date the supported extensions may not be correct.

I think we should be including all syntax in our own docs in a consistent manner.

@openshiftio/docs

**Comment by Preeticp** [comment]
@christianvogt, you are right, our docs should point to the official page here: https://access.redhat.com/help/markdown, but it was brought to the docs team notice that the extensions here are what a user needed to use, hence we had to point the users there. Please see here for discussion on this: https://github.com/openshiftio/openshift.io/issues/3065#issuecomment-382305397

@xcoulon, in this context I have two questions:
1. Do we have the syntax used by us listed in the developer documentation in the WIT repo? If so, can you share the link so that the docs could link to that page? If we do not have it in the developer docs, we might need to create a fresh reference doc for this in the docs repo. 
2. Out of the extensions listed here, can you please confirm if all of them are used by us or only a few, if the latter, which ones?

**Comment by serenamarie125** [comment]
Based on a conversation held on 2019-Jan-04 including Dev, PM and UX, the UX team will not be providing support for planner items. Closing this issue since this was solely around gathering requirements.
